src/NewCommand.php

```diff
@@ -635,7 +661,7 @@ protected function canResolveHostname($hostname)
     protected function getVersion(InputInterface $input)
     {
         if ($input->getOption('dev')) {
-            return 'dev-master';
+            return 'dev-slim-skeleton-11.x';
```

Comment:
```suggestion
            return 'dev-master';
```

This needs to be reverted, once the slim-skeleton branch is merged.